tests/test_kokkosfunctions_translator.py

```diff
@@ -165,7 +166,7 @@ def test_float_sum(self):
         expected_result: float = self.threads * (self.f_1 + self.f_2)
         result: float = pk.parallel_reduce(self.range_policy, self.functor.add_floats)
 
-        self.assertEqual(expected_result, result)
+        assert_allclose(expected_result, result)
```

Comment:
note that for the NumPy testing functions we should also really use the order `actual, expected`, though I believe the current convention in pykokkos does largely the reverse, so I didn't change that yet..

Reply:
Good point. Should we open another PR? Or do you want to fix it here?